app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,46 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def new
+  end
+
+  def create
+    merchant = Merchant.new({
+      name: params[:name],
+      address: params[:address],
+      city: params[:city],
+      state: params[:state],
+      zip: params[:zip],
+      })
+      merchant.save
+      redirect_to '/merchants'
+  end
```

Comment:
you can use your merchant params to create this merchant. 

```ruby 
def create 
   merchant = Merchant.create(merchant_params)
   redirect_to '/merchants'
end 
```

Using create instead of new will actually save it into your database.